config.go

```diff
@@ -98,6 +98,10 @@ const (
 	// HTLCs on our channels.
 	minTimeLockDelta = routing.MinCLTVDelta
 
+	// MaxTimeLockDelta is the maximum CLTV delta that can be applied to
+	// forwarded HTLCs.
+	MaxTimeLockDelta = routing.MaxCLTVDelta
```

Comment:
Do we need an extra alias here? Can't we just use `routing.MaxCLTVDelta` in the `rpcserver.go`?

Reply:
I just want to keep the style of `minTimeLockDelta`, because there is a line `minTimeLockDelta = routing.MinCLTVDelta` above in line `99`.